integration_tests/contracts/contracts/TestCRC20Proxy.sol

```diff
@@ -51,7 +51,7 @@ contract TestCRC20Proxy {
 
     function transfer_from_cronos_module(address addr, uint amount) public {
         require(msg.sender == module_address);
-        crc20Contract.transferFrom(module_address, addr, amount);
+        crc20Contract.transfer(addr, amount);
```

Comment:
in case of proxy contract, msg sender for the transfer is the proxy address, not module address